scripts/aws_sts.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Requesting a token code from the user
+read -p "Enter your aws user: " aws_user
+read -p "Enter your MFA authenticator name: " mfa
+read -p "Enter token code: " token_code
+
+# Get caller identity
+caller_identity=$(aws sts get-caller-identity)
+AccountID=$(echo "$caller_identity" | jq -r '.Account')
+
+# Command execution aws sts assume-role с подстановкой значения кода токена
+output_aws=$(aws sts assume-role --role-arn arn:aws:iam::"$AccountID":role/Admins --role-session-name "$aws_user"\
+ --serial-number arn:aws:iam::"$AccountID":mfa/"$mfa" --token-code "$token_code" --profile default)
```

Comment:
we need to add the ability to specify profile name here. I have different value for default profile

Reply:
@ggramal Corrected, you can check.